test_infra/src/lib.rs

```diff
@@ -589,3 +590,17 @@ pub fn ssh_command_ip(
         timeout,
     )
 }
+
+pub fn exec_host_command_status(command: &str) -> ExitStatus {
+    std::process::Command::new("bash")
+        .args(&["-c", command])
+        .status()
+        .expect(&format!("Expected '{}' to work", command))
+}
+
+pub fn exec_host_command_output(command: &str) -> Output {
+    std::process::Command::new("bash")
+        .args(&["-c", command])
+        .output()
+        .expect(&format!("Expected '{}' to work", command))
```

Comment:
You don't actually check the exit value here.

Reply:
Same here, I'll change the message.